content/en/infrastructure/containers/configuration.md

```diff
@@ -190,11 +190,61 @@ Set the environment variable on both the Process Agent and Cluster Agent contain
 {{% /tab %}}
 {{< /tabs >}}
 
+### Collect custom resources and CustomResourceDefinitions
+
+The [Orchestrator Explorer][3] collects CustomResourceDefinitions by default. These definitions appear in Datadog without any user configuration required.
+
+To collect custom resources, you need to configure both the Datadog Agent and set up indexing.
+
+1. Configure the Datadog Agent:
+
+   {{< tabs >}}
+   {{% tab "Helm Chart" %}}
+
+   Add the following configuration to `datadog-values.yaml`:
+
+   ```
+   orchestratorExplorer:
+       customResources:
+           - <CUSTOM_RESOURCE_NAME>
+   ```
+
+   Each `<CUSTOM_RESOURCE_NAME>` must use the format `group/version/kind`.
+
+   {{% /tab %}}
+   {{% tab "Datadog Operator" %}}
+
+   The Datadog Operator needs permission to collect custom resources. Install the Operator with an option that grants this permission:
+
+   ```
+   helm install datadog-operator datadog/datadog-operator --set clusterRole.allowReadAllResources=true
+   ```
+
+   Then, add the following configuration to `datadog-agent.yaml`:
```

Comment:
I would suggest calling this something else, so it's not confused with the Helm values file referenced above. (For added clarity, we can add a link to examples, like https://github.com/DataDog/datadog-operator/tree/main/examples/datadogagent) 

```suggestion
   Then, add the following configuration to your `DatadogAgent` manifest, `datadogagent.yaml`:
```

Reply:
I should note that it's called `datadog-agent.yaml` everywhere else in the docs, and the Helm values file is consistently called `datadog-values.yaml`. also, because of the tabbed structure, users would only see either the Operator or Helm tab at a time.

Reply:
Right, that's why I suggest to call it something different - it's not a Helm file in this case, but a Kubernetes manifest

Reply:
Ah gotcha, I'll change it!

Reply:
Great, thank you